src/parser/csql_lexer.l

```diff
@@ -1015,9 +1015,12 @@ IDL       [a-zA-Z0-9_]
 										csql_yylval.cptr = pt_makename(yytext);
 										return THAN; }
 [tT][iI][mM][eE]							{ begin_token(yytext);   return Time; }
-[tT][iI][nN][yY][iI][nN][tT]				{ begin_token(yytext);   return SmallInt; }
+[tT][iI][nN][yY][iI][nN][tT]				                { begin_token(yytext);   return SmallInt; }
```

Comment:
You need to register "tinyint" as a reserved word.

Reply:
No changes were made to the existing TINYINT keyword except for the indentation. 
The new keyword, TYPE, introduced in this PR was added to the derivations from Identifier and to the keyword list in keyword.c. 
Thank you.